src/registrar/tests/test_admin_request.py

```diff
@@ -263,6 +265,29 @@ def test_collaspe_toggle_button_markup(self):
         self.assertContains(response, domain_request.requested_domain.name)
         self.assertContains(response, "<span>Show details</span>")
 
+    @less_console_noise_decorator
+    def test_domain_requests_by_portfolio(self):
```

Comment:
Suggestion: Add a non-portfolio domain request and assert that it is filtered out